source/draw/RasterMonochrome.ooc

```diff
@@ -63,16 +75,11 @@ RasterMonochrome: class extends RasterPacked {
 		this resizeTo(size, InterpolationMode Smooth) as This
 	}
 	resizeTo: override func ~withMethod (size: IntVector2D, method: InterpolationMode) -> This {
-		result: This
-		if (this size == size)
-			result = this copy()
-		else {
-			result = This new(size)
-			match (method) {
-				case InterpolationMode Smooth => RasterPackedCanvas _resizeBilinear(this buffer pointer as ColorMonochrome*, result buffer pointer as ColorMonochrome*, this, result, IntBox2D new(this size), IntBox2D new(result size))
-				case => RasterPackedCanvas _resizeNearestNeighbour(this buffer pointer as ColorMonochrome*, result buffer pointer as ColorMonochrome*, this, result, IntBox2D new(this size), IntBox2D new(result size))
-			}
-		}
+		result := This new(size)
+		result canvas interpolationMode = method
+		result canvas draw(this, IntBox2D new(this size), IntBox2D new(size))
+		//TODO will be fixed by subcanvas
```

Comment:
Issue is already created for this: https://github.com/cogneco/ooc-kean/issues/857